04-advanced-topics/network-analysis_rnaseq_01_wgcna.Rmd

```diff
@@ -0,0 +1,456 @@
+---
+title: "WGCNA - RNA-seq"
+author: "CCDL for ALSF"
+date: "November 2020"
+output:   
+  html_notebook:
+    toc: true
+    toc_float: true
+    number_sections: true
+---
+
+# Purpose of this analysis
+
+In this example, we use weighted gene co-expression network analysis (WGCNA) to identify co-expressed gene modules [@Langfelder2008]. 
+WGCNA uses a series of correlations to determine what genes are expressed together in your data set. 
+This is a fairly intuitive approach to gene network analysis which is pretty widely used for microarray (as well as RNA-seq). 
+
+As output, WGCNA gives groups of co-expressed genes as well as an eigengene x sample matrix (where eigengenes contain the summarized variance for a group of co-expressed genes) [@Alter2000; @Langfelder2007]. 
```

Comment:
> their eigengenes are actually a bit different

I think this may be part of my problem interpreting things, I've seen the word eigengene used in several contexts but no clear definition stated overall. 

Thanks for the clarification.